app/assets/stylesheets/content/_badges.md

```diff
@@ -0,0 +1,13 @@
+# Badges
```

Comment:
I took the default Foundation name, but if someone can suggest a better name for this component, please do.